hnclient-v2/src/test/java/ca/bc/gov/hlth/hnclientv2/error/ErrorBuilderUnitTest.java

```diff
@@ -0,0 +1,46 @@
+package ca.bc.gov.hlth.hnclientv2.error;
```

Comment:
Is the name of this class just temporary until the original ErrorBuilderTest is removed?